js/views/modals/editListing/EditListing.js

```diff
@@ -18,6 +20,29 @@ import app from '../../../app';
 import BaseModal from '../BaseModal';
 import ShippingOption from './ShippingOption';
 
+$.trumbowyg.svgPath = '../node_modules/trumbowyg/dist/ui/icons.svg';
```

Comment:
Since this is a global setting, IMHO it shouldn't be made here, which is just one module using the editor.

I would probably make a wrapper module that does this - since modules are cached, it will only execute once:

```
// js/utils/trumbowyg.js
import 'trumbowyg';
$.trumbowyg.svgPath = '../node_modules/trumbowyg/dist/ui/icons.svg';
```

then any module / view that wants to use the plugin could import our wrapper:

```
import 'js/utils/trumbowyg';
```